engine/table/python/core/deephaven_jpy_init.py

```diff
@@ -18,188 +16,5 @@
 # If you want jpy to tell you about all that it is doing, change this
 # jpy.diag.flags = jpy.diag.F_ALL
 
-from deephaven import Config
-import deephaven.TableTools as ttools
-from deephaven.python_to_java import dataFrameToTable
-
-# NOTE: **THIS IS REQUIRED** for WorkerPythonEnvironment - don't take it out...
-import __main__
-
-
-###############################################################################################
-
-# can't use IsWidget.get_dh_table or the data frame will be interpreted as a table instead of a widget?
-pandas.DataFrame.to_dh_table = lambda self: dataFrameToTable(self, True)
-
-
-########################################################################################################
-# Performance monitoring
-
-def add_tables_from_map_to_binding(tables):
-    """
-    Iterates through a (java) map of tables and adds them to the global binding. This is a helper method
-    accommodating for the fact that the performance queries return the table results as a (java) HashMap object.
-
-    :param tables: java HashMap
-    """
-
-    it = tables.entrySet().iterator()
-    while it.hasNext():
-        pair = it.next()
-        globals()[pair.getKey()] = pair.getValue()
-        it.remove()
-
-
-def query_update_performance_set(evaluationNumber):
-    """
-    Name matched to Groovy equivalent for coherent usage.
-    TODO: document this
-
-    :param worker_name:
-    :param date:
-    :param use_intraday:
-    :param server_host:
-    :param as_of_time:
-    :return:
-    """
-
-    _jtype_ = jpy.get_type("io.deephaven.engine.table.impl.util.PerformanceQueries")
-
-    tableMap = _jtype_.queryUpdatePerformanceMap(evaluationNumber)
-    add_tables_from_map_to_binding(tableMap)
```

Comment:
If we are taking this out, we need a ticket to wrap `io.deephaven.engine.table.impl.util.PerformanceQueries`.  In enterprise, these are queries that make the perf analysis of queries much easier.  I assume the same may still be true in DHC.

Reply:
the new perfmon module in #1949 wraps it and that's why I removed it.